pkg/graph/schema.graphql

```diff
@@ -37,6 +37,40 @@ type RequestEdge {
   node: Request!
 }
 
+"""
+CedarAuthorityToOperate represents the response from the /authorityToOperate endpoint from the CEDAR Core API.
+"""
+type CedarAuthorityToOperate {
+  cedarId: String!
+  uuid: String!
+  actualDispositionDate: Time
+  containsPersonallyIdentifiableInformation: Boolean
+  countOfTotalNonPrivilegedUserPopulation: Int
```

Comment:
Because this (and the following two count fields) are counts, should they be non-nullable in GQL and default to 0? The CEDAR API returns the number 0 for them; I'm not quite sure what we should do. IMO if CEDAR returning 0 means that they know for sure there's 0 non-privileged users/open POAMs/privileged users, we should return 0; if it means they don't know, we should return null.

@ClayBenson94 (this might be something to ask the CEDAR team)

Reply:
John Shoup [just said on Slack](https://cmsgov.slack.com/archives/C02KTCN3ADD/p1652983526196749?thread_ts=1652982592.922789&cid=C02KTCN3ADD) that they get a value of 0 from CFACTS, so that's probably what our backend should return; don't think we need to make any changes.